website/docs/add-secure-apps/flows-stages/stages/user_logout.md

```diff
@@ -2,4 +2,29 @@
 title: User Logout stage
 ---
 
-Opposite stage of [User Login Stages](./user_login/index.md). It removes the user from the current session.
+The User Logout stage terminates the user's session in authentik and, if configured, triggers [Single Logout](./single_logout.md) in [SAML](../../providers/saml/saml_single_logout.md) and [OIDC](../../providers/oauth2/fontchannel_and_backchannel_logout.mdx) providers.
+
+## Logout flow injection
+
+authentik dynamically injects logout stages into the user logout flow when provider sessions requiring logout are detected:
```

Comment:
@Connor do you mean the User Logout stage... I don't think we have a flow named that?

Reply:
Also, this statement applies to situations when SLO is not configured, correct?

Reply:
User's current logout flow...? either that or your suggestion, thanks